padertorch/configurable.py

```diff
@@ -892,12 +951,22 @@ def recursive_class_to_str(config, sort=False):
         return config
 
 
-def _split_factory_kwargs(config):
+def _split_factory_kwargs(config, key='factory'):
     kwargs = config.copy()
-    factory = kwargs.pop('factory')
+    factory = kwargs.pop(key)
     return factory, kwargs
 
 
+def _get_special_key(config):
+    # These special keys are used in the config to indicate classes or
+    # functions 'factory' is used to specify initialized classes or
+    # function outputs as an input. 'partial' is used if an input is a
+    # non-initialized class or a functions
+    for key in ['factory', 'partial']:
+        if key in config.keys():
+            return key
```

Comment:
Add a verbose `return None`? i.e. highlight that the return value is one of: `'factory'`, `'partial'`, `None`